crates/telio-starcast/src/lib.rs

```diff
@@ -5,6 +5,15 @@
 //! 2. Starcast transport component responsible for multicasting intercepted traffic to the meshnet peers and handling response messages.
 //! 3. Nat is transport's utility used to SNAT the packets comming from multiple peers into one multicast peer address.
 
+use const_format::formatcp;
+
+pub(crate) const IPV4_MULTICAST_NETWORK: &str = "224.0.0.0/4";
```

Comment:
We had these constant in telio-model::constants, i would prefer to keep than grouped there so it would be clear what global ip's are reserved